website/index.html

```diff
@@ -390,13 +390,6 @@
             to take part in this cyber security
             competition. You must also be registered for HackPSU to participate.
           </p>
-          <div class="ctf-register-btn">
-            <a href="https://ctf.hackpsu.org">
-              <div class="header-text text-center">
-                <p class="text-center ctf-register">HackPSU CTF Sign Up</p>
```

Comment:
I think ethan shut down the ctf server so it send you to login to somewhere idk

Reply:
assuming we'll be doing CTF again it might be worth just commenting this out for less work next semester

Reply:
True but at the same time it could be a good intro task for a new member, we could always come back to this PR and bring it back tho